src/themes/default/components/core/blocks/Header/Header.vue

```diff
@@ -172,20 +172,33 @@ export default {
   .center-xs {
     text-align: center;
   }
-  @media (max-width: 767px) {
-    .row.middle-xs {
+  .row.middle-xs {
+    
+    @media (max-width: 767px) {
       padding-left: 0;
+      padding-right: 0;
       &.py5 {
         margin: 0;
       }
     }
-    .col-xs-2:first-of-type {
-        padding-left: 0;
+    @media (max-width: 374px) {
+      padding-right: 15px;  
     }
-    .col-xs-2:last-of-type {
-        padding-right: 15px;
+  }
+
+  .col-xs-2:first-of-type {
```

Comment:
We should not styling by grid classes. We should add some additional class.

Reply:
https://github.com/DivanteLtd/vue-storefront/blob/master/src/themes/default/components/core/blocks/Header/Header.vue#L183
I just fix this to fit it in new grid.
Should I rewrite this normally in this PR?

Reply:
@mercs600 anyway, i added some classes